docs/user-guide/testing/integration_tests.md

```diff
@@ -1,3 +1,11 @@
 # Integration Tests
 
 Coming soon
+
+## Running the load tests
```

Comment:
I think we need a bit of info on what the load tests do, and how we have set them up.

Reply:
Done in 7bf27e596ff00146cd09be72f3727552c80025f4